### DISCUSSION TITLE
the temperature is so high compared to windows system

### DISCUSSION THREAD

**Discussion by AxtonJi** [badstorm/surface-pro-7-opencore#80] [Q&A]
even when I watch some videos, or play a game for kids that only occupies 50mb, after about 5 minutes,  the machine gets incredibly hot. Do you meet with such problems? What can I do?
besides, when I turn on the microphone or starts this Mac OS,there is a loud noise. I think there is something wrong with the applealc.kext.

**Comment by badstorm** [comment]
Since is not a real bug (in my case i don't have this issue) I move your question to discussion. So I other users have this problem can you discuss there.

**Comment by owenthehugger** [comment]
Same here
